<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>737</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>737</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
    <underline>false</underline>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string> </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #36393F;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Ваше место для просмотра погоды</string>
    </property>
   </widget>
   <widget class="QPushButton" name="begin_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-size: 24px;
	color: #fff;
	background: #4A77F2;
	text-decoration: none;
	border-radius: 20px;
	transition: .5s;
}
QPushButton:hover{
	color: #4A77F2;
	background: #FFF;
}
QPushButton:pressed{
	background: #CCCCCC;
  	box-shadow: 0 3px rgb(33,147,90) inset;
	
}</string>
    </property>
    <property name="text">
     <string>начать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Сoncord Weather - простейший способ ежедневного просмотра погоды для любого
города: Москва, Санкт-Петербург, Сан-Франциско, Голливуд и даже Ханты-Мансийск.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string>Фирсов Кирилл все права защищены©</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
